.gitlab-ci.yml

```diff
@@ -801,6 +801,12 @@ library:ci-jasmin:
   - library:ci-mathcomp
   - library:ci-mathcomp_word
 
+library:ci-coq-library-undecidability:
+  extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - plugin:ci-metacoq
```

Comment:
you need equations too
and equations/metacoq don't use flambda

Reply:
the dependencies need all the transitive deps, so all of build:base, equations and metacoq

Reply:
done